otseobande/hackerbay-backend
# #16219593 Implement login

**What does this PR do?**

This PR implements a login route on the application

**Description of Task to be completed?**

- install express
- create controller to handle login
- create middleware to validate login
- add login route to the application


**How should this be manually tested?**

- Fetch and checkout to this branch
- Open Postman and navigate send a post request with a json body containing username and password to `localhost:8000/api/v1/auth/login`
- You should receive a response with a jwt token

**Any background context you want to provide?**

N/A

**What are the relevant pivotal tracker stories?**

-  [#16219593](https://www.pivotaltracker.com/story/show/162195933)

**Screenshots (if appropriate)**

<img width="1281" alt="screen shot 2018-11-26 at 6 16 07 pm" src="https://user-images.githubusercontent.com/17123317/49030352-602a0880-f1a7-11e8-9324-13ecd677170b.png">
